### ISSUE TITLE
Exception: column "values" does not exist (PQ::PQError)

### ISSUE THREAD

**Issue by tmiland** [omarroth/invidious#390]
Get this error on latest Master:

`/feed/subscriptions`

> Looks like you've found a bug in Invidious. Feel free to open a new issue here or send an email to omarroth@protonmail.com. 

From log:

```
2019-03-02 21:45:44 +01:00 500 GET /feed/subscriptions 136.13ms
Exception: column "values" does not exist (PQ::PQError)
  from lib/pg/src/pq/connection.cr:157:7 in 'read'
  from lib/pg/src/pq/connection.cr:152:7 in 'expect_frame'
  from lib/pg/src/pg/statement.cr:19:5 in 'query_all:as'
  from src/invidious.cr:1966:20 in '->'
  from lib/kemal/src/kemal/route.cr:255:3 in '->'
  from lib/kemal/src/kemal/route_handler.cr:255:3 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:24:7 in 'call'
 
 from /usr/share/crystal/src/http/server/handler.cr:24:7 in 'call_next'
  from lib/kemal/src/kemal/filter_handler.cr:21:7 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:24:7 in 'call_next'
  from /usr/share/crystal/src/http/server/handler.cr:24:7 in 'call_next'
  from /usr/share/crystal/src/http/server/handler.cr:24:7 in '???'
  from /usr/share/crystal/src/http/server/handler.cr:24:7 in '???'
  from lib/kemal/src/kemal/static_file_handler.cr:67:9 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:24:7 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:24:7 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:24:7 in 'call'
  from /usr/share/crystal/src/http/server/request_processor.cr:39:11 in 'process'
  from /usr/share/crystal/src/fiber.cr:255:3 in '???'
  from ???
```

**Comment by tmiland** [comment]
Error occurs when:

- [X] Only show unwatched:

Is ticked.

**Comment by omarroth** [comment]
Should be fixed with 03f94db5e2541cb9474d90737d9c0ae95609835a.

**Comment by tmiland** [comment]
> Should be fixed with [03f94db](https://github.com/omarroth/invidious/commit/03f94db5e2541cb9474d90737d9c0ae95609835a).

It is 👍
